scripts/perftests/PerformanceTestUtilities.ps1

```diff
@@ -479,9 +484,49 @@ Function RunRestore(
         $staticGraphOutputValue = "N/A"
     }
 
+    If($isClientDotnetExe -Or $isClientMSBuild)
+    {
+        # Always disable NuGetAudit. It's something impacted by the time of execution, and not helpful in these type of performance tests.
+        $arguments.Add("/p:NuGetAudit=false")   
+    }
+
+    if(($isClientDotnetExe -Or $isClientMSBuild) -And $forceLegacyResolverFallback)
+    {
+        $arguments.Add("/p:RestoreUseLegacyDependencyResolver=true")
+    }
+
+    if($useLocallyBuiltNuGet)
+    {
+        if($isClientMSBuild)
+        {
+            $NETFramework = "net472"
+            $Configuration = "release"
+            $packDllPath = Join-Path $NuGetClientRoot "artifacts\NuGet.Build.Tasks.Pack\bin\$Configuration\$NETFramework\NuGet.Build.Tasks.Pack.dll"
+            $packTargetsPath = Join-Path $NuGetClientRoot "src\NuGet.Core\NuGet.Build.Tasks.Pack\NuGet.Build.Tasks.Pack.targets"
+            $restoreDllPath = Join-Path $NuGetClientRoot "artifacts\NuGet.Build.Tasks\bin\$Configuration\$NETFramework\NuGet.Build.Tasks.dll"
+            $nugetRestoreTargetsPath = Join-Path $NuGetClientRoot "src\NuGet.Core\NuGet.Build.Tasks\NuGet.targets"
+            $nugetPropsPath = Join-Path $NuGetClientRoot "src\NuGet.Core\NuGet.Build.Tasks\NuGet.props"
+            $consoleExePath = Join-Path $NuGetClientRoot "artifacts\NuGet.Build.Tasks.Console\bin\$Configuration\$NETFramework\NuGet.Build.Tasks.Console.exe"
+            $arguments.Add("/p:NuGetRestoreTargets=$nugetRestoreTargetsPath");
+            $arguments.Add("/p:NuGetPropsFile=$nugetPropsPath");
+            $arguments.Add("/p:RestoreTaskAssemblyFile=$restoreDllPath");
+            $arguments.Add("/p:NuGetBuildTasksPackTargets=$packTargetsPath");
+            $arguments.Add("/p:NuGetConsoleProcessFileName=$consoleExePath");
+            $arguments.Add("/p:ImportNuGetBuildTasksPackTargetsFromSdk=true");
+            $arguments.Add("/p:NuGetPackTaskAssemblyFile=$packDllPath");
```

Comment:
I expect this is also set by the targets, so as long as the targets file is being set to the local build location, setting this property is unnecessary.

Reply:
The structure of the output folder doesn't match the structure that the pack targets have in the SDK or the package itself, so this is necessary.